common/gopool/pool.go

```diff
@@ -9,7 +9,7 @@ import (
 
 var (
 	// Init a instance pool when importing ants.
-	defaultPool, _   = ants.NewPool(ants.DefaultAntsPoolSize, ants.WithExpiryDuration(10*time.Second))
+	defaultPool, _   = ants.NewPool(ants.DefaultAntsPoolSize, ants.WithExpiryDuration(time.Hour))
```

Comment:
Why 1 hour here?

Reply:
If you don't like the idea of expiration tasks in 1 global pool, remove the expiry option in the global pool, create a volatile pool without any options will be an acceptable solution.

Reply:
use `ants` version not exist `No Expiry` option, need upgrade version to >=2.6.0, I'm try upgrade 2.8.0  [e12de3c](https://github.com/dogechain-lab/dbsc/pull/27/commits/e12de3cbd2f9a46889da3cd99741d29011078713)